scripts/octoeverywhere.sh

```diff
@@ -273,9 +273,51 @@ function remove_octoeverywhere()
 #===================================================#
 
 function update_octoeverywhere() {
-  # Since our update might require new python packages or system packages, ask the user to use the
-  # moonraker update manager to do it, since that takes care of it.
-  print_error "Please use the Moonraker Update Manager from the Mainsail or Fluidd UI to update OctoEverywhere.\n Contact our support team if you need help! support@octoeverywhere.com"
+  do_action_service "stop" "octoeverywhere"
+
+  if [[ ! -d ${OCTOEVERYWHERE_DIR} ]]; then
+    clone_moonraker "${OCTOEVERYWHERE_DIR}"
+  else
+    backup_before_update "octoeverywhere"
+    status_msg "Updating OctoEverywhere for Klipper ..."
+    cd "${OCTOEVERYWHERE_DIR}" && git pull
+    ### read PKGLIST and install possible new dependencies
+    install_octoeverywhere_dependencies
+    ### install possible new python dependencies
+    "${OCTOEVERYWHERE_ENV}"/bin/pip install -r "${OCTOEVERYWHERE_DIR}/requirements.txt"
```

Comment:
I don't know much about virtual environments, but I assume this installs into it? The only way I have interactive with virtual envs is by using the /bin/activate the script.

Reply:
Correct, i added the evn you create during setup to the global variables and with that command i just run pip from that env and install possible new python deps into that specific env.